src/v/storage/mvlog/tests/file_test.cc

```diff
@@ -35,12 +35,17 @@ class FileTest : public seastar_test {
     ss::future<> SetUpAsync() override {
         co_await ss::recursive_touch_directory(base_dir_);
     }
+
     ss::future<> TearDownAsync() override {
         for (auto& file : files_) {
             co_await file->close();
         }
-        co_await ss::recursive_remove_directory(
-          std::filesystem::path{base_dir_});
+        // NOTE: death tests may clean themselves up in the test body to avoid
```

Comment:
"may clean" or "may need to clean"?

Reply:
Yeah it should be "need to". Can follow up if you think it's unclear enough to trip up on